perl/p5-tkx/Portfile

```diff
@@ -20,7 +21,6 @@ checksums           rmd160  d9901b5c3c033a49003331ff04c9d152746edc2b \
 
 if {${perl5.major} != ""} {
     depends_lib-append \
+                    port:tk \
                     port:p${perl5.major}-tcl
-
-    livecheck.type  none
```

Comment:
The idea is to only have the livecheck in the non-subport to avoid listing it twice for the maintainer. This also seems unrelated to the other change, so I would not combine them in the same commit anyway.

Reply:
Removed in f148a36030. Would something like that then belong in the perl5 PortGroup?